windows-msi/msi.wxs

```diff
@@ -925,7 +925,7 @@
                 Id="OpenVPN.Service"
                 Title="$(var.PRODUCT_NAME) Service"
                 Description="Service wrappers"
-                Level="2"
+                Level="3"
```

Comment:
Currently we install this by default, dont we?

Reply:
True, but it would also mean I need to set the `openvpnserv2.exe` service start to OnDemand/Manual - to keep consistent experience with NSIS installations.

But, I am afraid users needing this service will always get this service reset to Manual start after updating OpenVPN.

There is a subtle difference between NSIS and MSI... In MSI we could query the configured service start type in registry like NSIS does. Unfortunately, we cannot use that property in MSI's `ServiceInstall` table, as the `StartType` column is `DoubleInteger`, not `Formatted` type. :(

In order to preserve the service start configuration over updates we would need quite some hacking (runtime MSI database manipulation) and/or some additional custom actions.

I made a note to research whether MSI resets the service start type when the `openvpnserv2.exe` component is updated.

Reply:
Update: `openvpnserv2.exe` service is already set to OnDemand start in MSI package. I have set it to install by default now to provide a more seamless transition from NSIS installers.